packages/openapi3/test/security.test.ts

```diff
@@ -521,4 +521,72 @@ worksFor(["3.0.0", "3.1.0"], ({ diagnoseOpenApiFor, openApiFor }) => {
       },
     ]);
   });
+
+  it("deduplicates OAuth2 scopes across multiple flows", async () => {
```

Comment:
This is a fine test to have in the `@typespec/openapi3` package. However, the change we made is in the `@typespec/http` package. Can you add a test to the `auth.test.ts` file in the `@typespec/http` package that confirms that oauth `scopes` don't contain duplicate scopes?